theano/tensor/tests/test_mpi.py

```diff
@@ -45,15 +46,21 @@ def test_mpi_roundtrip():
     if not mpi_enabled:
         raise SkipTest('MPI not enabled')
     theano_root = theano.__file__.split('__init__')[0]
+    d = {}
+    if PY3:
+        # Is some not understood cases, the subprocess never finish.
```

Comment:
As you have now added `p.communicate()` below, does the subprocess still not finish ?

Reply:
I don't know. We only get that in the daily buildbot. I prefer to fix all the cases we want.

Also, jenkins got broken due to system update... So I'm not sure if futur case where it pass would have this fixed, as I'm not sure the new install will reproduce the same error.

Reply:
ok